MANIFEST.in

```diff
@@ -0,0 +1,2 @@
+include *.txt *.ini *.cfg *.rst
+recursive-include testapp *.ico *.png *.css *.gif *.jpg *.pt *.txt *.mak *.mako *.js *.html *.xml
```

Comment:
You should add your README file to this list by adding *.md to the `include` line.